pkgs/by-name/mu/muffon/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchurl
+, appimageTools
+, makeWrapper
+}:
+
+let
+  pname = "muffon";
+  version = "2.0.2";
+  src = fetchurl {
+    url = "https://github.com/staniel359/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
+    hash = "sha256-bJfnYX5ZCAJNSGP4WJcDpVKX7yAqFYC7sqDIGQWwkTM=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname src version; };
+
+in appimageTools.wrapType2 {
+  inherit pname src version;
+
+  extraInstallCommands = ''
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
+    source "${makeWrapper}/nix-support/setup-hook"
+    wrapProgram $out/bin/${pname} \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
+    install -m 444 -D ${appimageContents}/muffon.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/muffon.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
```

`--replace` is deprecated. Consider using `--replace-fail` or `--replace-warn`
```suggestion
      --replace-fail 'Exec=AppRun' 'Exec=${pname}'
```